lib/trento/domain/sap_system/sap_system.ex

```diff
@@ -88,7 +89,18 @@ defmodule Trento.Domain.SapSystem do
     embeds_one :application, Application
   end
 
+  def execute(
+        %SapSystem{},
```

Comment:
Shouldn't we match with `%SapSystem{sap_system_id: nil},` ? otherwise, what happens with sap systems with primary instances registered?

Reply:
done man thanks